ceph_deploy/util/init_systems.py

```diff
@@ -0,0 +1,246 @@
+from ceph_deploy.lib import remoto
+
+# Provide mappings/translations for services depending
+# on Init system.  All services coming into the classes
+# here should be one of ceph-mon, ceph-osd, ceph-mds, or
+# ceph-radosgw. It's then up to the code here to Do The
+# Right Thing.
+SYSV_SERVICES = {
+    'ceph-mon': 'mon',
+    'ceph-mds': 'mds'
+}
+
+UPSTART_SERVICES = {
+    'ceph-radosgw': 'radosgw'
+}
+
```

Comment:
I think that SYSV_SERVICES and  UPSTART_SERVICES mappings are in the wrong place here.

These are distribution specific mappings, rather than init system specific.

This would change the interface a little but will make the code more neutral to SUSE, REDHAT, debian differences etc.